scripts/pre_commit_linter.py

```diff
@@ -1731,6 +1774,11 @@ def _check_for_copyright_notice(all_files):
     return summary_messages
 
 
+def _print_summary_messages():
```

Comment:
Rename this to `_print_complete_summary_of_errors()` or something more concrete since summary messages are being printed in almost all checks.

Reply:
done